src/components/Dropdown/V2/Dropdown.Container.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { Provider } from 'unistore/react'
+import { DropdownProps } from './Dropdown.types'
+import createStore, { initialState } from './Dropdown.store'
+import Dropdown from './Dropdown'
+import Menu from './Dropdown.Menu'
+import Item from './Dropdown.Item'
+import { pathResolve } from './Dropdown.utils'
+import Trigger from './Dropdown.Trigger'
+import { createUniqueIDFactory } from '../../../utilities/id'
+import { noop } from '../../../utilities/other'
+
+export interface Props extends DropdownProps {
+  // Secret prop to pass in a custom store
+  __store?: any
+  subscribe: (state: any) => void
+}
+
+export interface State {
+  id: string
+}
+
+const uniqueID = createUniqueIDFactory('hsds-dropdown-v2-')
+
+export class DropdownContainer extends React.PureComponent<Props, State> {
+  static defaultProps = {
+    ...initialState,
+    onBlur: noop,
+    onOpen: noop,
+    onClose: noop,
+    onFocus: noop,
+    isOpen: false,
+    innerRef: noop,
+    onSelect: noop,
+    subscribe: noop,
+    trigger: 'Dropdown',
```

Comment:
I wonder if `Dropdown` is the most useful default Trigger value, or if it should be `Open` or `Close` depending on the state.

Reply:
Gave this some thought! I think `Dropdown` is the easiest. Showing labels like `Open`/`Close` would mean they would need to specify 2 props (for open/close).

Ideally.. you'd always change what the Trigger is 😅 . (Be it the label or markup)
Hope that makes sense!

Reply:
Yeah, that's true. I would expect people to supply their own too. 👍 on your decision.